ldelf/ta_elf_rel.c

```diff
@@ -131,10 +131,12 @@ static void e32_process_dyn_rel(const Elf32_Sym *sym_tab, size_t num_syms,
 	size_t name_idx = 0;
 
 	sym_idx = ELF32_R_SYM(rel->r_info);
-	assert(sym_idx < num_syms);
+	if (sym_idx >= num_syms)
```

Comment:
Confirm `>=`, not `>` ?  (Indices start from 1?)

Reply:
The last valid sym_idx is num_syms - 1.